### DISCUSSION TITLE
does superset support multi-lang?

### DISCUSSION THREAD

**Discussion by Jimmy-Cheong** [apache/superset#17485] [Q&A / Help] [answered]
I did set my default lang to zh, but it did not work. 

![image](https://user-images.githubusercontent.com/10903558/142348012-1220326b-6282-45c7-bd98-5a15e37705bb.png)

and there is no language switch

![image](https://user-images.githubusercontent.com/10903558/142348362-aa5ebef0-9801-4de2-bdc7-d5574feab718.png)

**Comment by junlincc** [comment]
@stormszhang https://github.com/apache/superset/issues/2242

**Comment by 1c7** [comment, marked as answer]
https://github.com/apache/superset/discussions/18806  

<img width="953" alt="image" src="https://user-images.githubusercontent.com/1804755/154647315-2f9d66a1-e2a2-4099-b6ac-27cc3e5c64d6.png">

**Comment by tphundey** [comment]
hello sh`` 

> _`#[](url)`_
